lib/src/types.ts

```diff
@@ -1,19 +1,50 @@
 import ow from 'ow';
-import KeyPair from './key_pair';
-
-export enum Network {
-    Mainnet = 'Mainnet',
-    Testnet = 'Testnet',
-    Devnet = 'Devnet',
-}
-
-export const CustomTypes = {
-    KeyPair: ow.object.validate((value) => ({
-        validator: value instanceof KeyPair,
-        message: `Expected value to be an instance of KeyPair, got ${value.constructor.name}`,
-    })),
-    Network: ow.string.validate((value) => ({
-        validator: value === 'Mainnet' || value === 'Testnet' || value === 'Devnet',
-        message: `Expected value to be one of the network variants (Mainnet, Testnet, Devnet)`,
-    })),
+import BigNumber from 'bignumber.js';
+
+import { Network } from './network';
+import { MAX_COIN_BN, MAX_COIN_FORMATTED } from './init';
+
+const native = require('../../native');
+
+export const owBigNumber = ow.object.validate((value: object) => ({
+    validator: BigNumber.isBigNumber(value),
+    message: 'Expected value to be a BigNumber',
+}));
+
+export const owCoin = ow.object.validate((value: object) => ({
+    validator:
+        BigNumber.isBigNumber(value) &&
+        value.isInteger() &&
+        value.isGreaterThanOrEqualTo(0) &&
+        value.isLessThanOrEqualTo(MAX_COIN_BN),
+    message: `Expected value to be within maximum coin: ${MAX_COIN_FORMATTED}`,
+}));
+
+const validateTransferAddress = (value: string): boolean => {
+    let network: Network;
+    if (value.startsWith('cro')) {
```

Comment:
Do you think it's a good idea to extract `cro` `tcro` `dcro` out to be global constants (maybe put into `init.ts`)? Or it's too tedious to do so and will never change them anyway.

Reply:
Yes, I agree. this is achieved in the network constants PR.

Reply:
https://github.com/crypto-com/cro-nodelib/pull/13/files#diff-fe49f986da6c36ca628640d3d320a70cR27